veracruz-test/src/main.rs

```diff
@@ -56,8 +56,7 @@ mod tests {
     const STRING_1_DATA: &'static str = "hello-world-1.dat";
     const STRING_2_DATA: &'static str = "hello-world-2.dat";
 
-    use actix_rt::System;
-    use async_std::task;
+    use actix_rt::{ System, time::sleep };
```

Comment:
Weird formatting.  Has `rustfmt` been run on the file?

Reply:
Probably not

Reply:
Done